go/vt/vttablet/tabletmanager/tm_state.go

```diff
@@ -181,7 +181,24 @@ func (ts *tmState) ChangeTabletType(ctx context.Context, tabletType topodatapb.T
 		// Update the tablet record first.
 		_, err := topotools.ChangeType(ctx, ts.tm.TopoServer, ts.tm.tabletAlias, tabletType, PrimaryTermStartTime)
 		if err != nil {
-			return err
+			log.Errorf("error in changing type in tablet record in topo for tablet %s :- %v", topoproto.TabletAliasString(ts.tm.tabletAlias), err)
+			log.Errorf("will keep trying to read from the topo server")
```

Comment:
This can be combined into the previous log line with a comma or something similar separating the two parts of the error message.

Reply:
Fixed